chat/prekey/key_test.go

```diff
@@ -72,3 +73,10 @@ func TestPreKey_ToProtobufAndBack(t *testing.T) {
 	require.Equal(t, hex.EncodeToString(pp.Key), hex.EncodeToString(k.PublicKey[:]))
 
 }
+
+func TestPreKey_OlderThen(t *testing.T) {
+	k := PreKey{
+		time: time.Now().Truncate(time.Second * 10),
+	}
+	require.False(t, k.OlderThen(time.Second*5))
```

Comment:
OlderThen should be renamed to OlderThan to be grammatically correct.

Reply:
Done